primitives/runtime/src/lib.rs

```diff
@@ -110,3 +111,9 @@ pub trait Size {
 	/// accurate.
 	fn size_hint(&self) -> u32;
 }
+
+impl Size for usize {
```

Comment:
That implementation might be a bit misleading, no? The trait documentation says the function should return "approximate size of this object in bytes", not perform a `usize -> u32` conversion.

Is it only used for tests? Perhaps maybe better to have a wrapper type?